Jellyfin.Networking/Manager/INetworkManager.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.NetworkInformation;
+using MediaBrowser.Model.Configuration;
+using NetworkCollection;
+
+namespace Jellyfin.Networking.Manager
+{
+    /// <summary>
+    /// Interface for the NetworkManager class.
+    /// </summary>
+    public interface INetworkManager
+    {
+        /// <summary>
+        /// Event triggered on network changes.
+        /// </summary>
+        event EventHandler NetworkChanged;
+
+        /// <summary>
+        /// Gets the Published server override list.
+        /// </summary>
+        Dictionary<IPNetAddress, string> PublishedServerOverrides { get; }
```

Comment:
What does the value represent?

Reply:
That is the variable that holds the --published-server-url for specific ip ranges. If you've had to ask, then it needs a better description.

<PublishedServerUriBySubnet>
<string>192.168.1.0/24=net1access:1270</string>
<string>192.168.2.1/24=net2access</string>
<string>external=jellyfinoutside</string>
</PublishedServerUriBySubnet>